index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mint's Sign Up Page</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+</head>
+<body>
+    <div id="main-wrapper">
+        <header>
+            <div id="signin-link">
+                <div id="forgot-pwd">
+                    <span>Already have an Intuit account?</span>
+                    <span><a href="#">I forgot my User ID or Password</a></span>
+                </div>
+                <div id="signin-btn">
+                    <a href="#">
+                        <button id="signin">
+                        <i class="fas fa-lock"></i>
+                        Sign In
+                        </button>
+                    </a>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div id="header">
+                <div id="brand">
+                    <img id="brand-name" src="" alt=""> INTUIT
+                </div>
+                <div id="links">
+                    <a href="" class="brand-links">
+                        <img id="mint-svg" class="svg-brand-link" src="https://accounts.intuit.com/IUS-Plugins/v2/images/mint-favicon.svg" alt="mint">
+                        <img id="mint-txt" class="svg-brand-txt" src="https://accounts.intuit.com/IUS-Plugins/v2/images/mint-text.svg" alt="">
+                    </a>
+                    <a href="" class="brand-links">
+                        <img id="quickbooks-svg" class="svg-brand-link" src="https://accounts.intuit.com/IUS-Plugins/v2/images/quickbooks-favicon@2x.svg" alt="quickbooks">
+                        <img id="qb-txt" class="svg-brand-txt" src="https://accounts.intuit.com/IUS-Plugins/v2/images/quick-books-text.svg" alt="">
+                    </a>
+                    <a href="" id="turbotax-svg" class="brand-links">
+                        <img class="svg-brand-link" src="https://accounts.intuit.com/IUS-Plugins/v2/images/tt-favicon@2x.svg" alt="turbotax">
+                        <img id="turbo-txt" class="svg-brand-txt" src="https://accounts.intuit.com/IUS-Plugins/v2/images/turbo-tax-text.svg" alt="">
+                    </a>
+                </div>
+            </div>
+            <div id="form-wrapper">
+                <form action="" method="POST">
```

Comment:
Bad value for attribute action on element form: Must be non-empty.

Reply:
Added # value to action attribute on element form